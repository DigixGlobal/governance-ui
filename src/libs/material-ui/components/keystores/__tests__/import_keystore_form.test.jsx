import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import ImportKeystoreForm from '../import_keystore_form';

describe('<ImportKeystoreForm />', () => {
  test('renders correctly', () => {
    const component = shallow(
      <ImportKeystoreForm
        setError={jest.fn}
        setLoading={jest.fn}
        onGetPrivateKey={jest.fn}
      />,
    );

    expect(toJson(component)).toMatchSnapshot();
  });
});
