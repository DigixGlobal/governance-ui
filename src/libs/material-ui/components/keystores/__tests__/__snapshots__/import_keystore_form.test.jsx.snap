// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ImportKeystoreForm /> renders correctly 1`] = `
<Dropzone
  activeClassName="active"
  className="ui message info dragbox"
  disableClick={false}
  disablePreview={false}
  disabled={false}
  maxSize={Infinity}
  minSize={0}
  multiple={true}
  onDrop={[Function]}
  preventDropOnDocument={true}
>
  <div
    className="ui padded one column grid"
  >
    <div
      className="center aligned column"
    >
      <i
        className="icon upload large"
      />
      <p>
        Click or drag and drop keystore here to upload.
      </p>
    </div>
  </div>
</Dropzone>
`;
